doc/dev_guide/building_from_source.rst

```diff
@@ -5,279 +5,226 @@
-     4.6 or later. On Mac OS X, this is ``Clang`` version 3.2+.
+    *   A C/C++ compiler.
 
-   * A program for managing the build process. |br| For all platforms, this is
-     ``CMake`` version 2.8+.
+        Ordinarily, this is ``gcc`` and ``g++`` version 4.6 or later.
+        On Mac OS X, this is ``Clang`` version 3.2+.
 
-   * A build automation tool. |br| For all platforms this is ``GNU Make``.
+    *   cmake
 
-   * `ReadLine <http://www.gnu.org/software/readline/>`_ library, any version
-   * `ncurses <https://www.gnu.org/software/ncurses/>`_ library, any version
-   * `OpenSSL <https://www.openssl.org>`_ library, version 1.0.1+
-   * `ICU <http://site.icu-project.org/download>`_ library, recent version
-   * `Autoconf <https://www.gnu.org/software/autoconf/>`_ library, any version
-   * `Automake <https://www.gnu.org/software/automake/>`_ library, any version
-   * `Libtool <https://www.gnu.org/software/libtool/>`_ library, any version
-   * `Zlib-devel <https://www.zlib.net/>`_ library, any version
+    *   autoconf automake libtool
 
-   * Python and modules. |br| Python interpreter is not necessary for building
-     Tarantool itself, unless you intend to use the "Run the test suite"
-     option in step 5. For all platforms, this is ``python`` version 2.7+
-     (but not 3.x). You need the following Python modules:
+    *   make
 
-       * `pyyaml <https://pypi.python.org/pypi/PyYAML>`_ version 3.10
-       * `argparse <https://pypi.python.org/pypi/argparse>`_ version 1.1
-       * `msgpack-python <https://pypi.python.org/pypi/msgpack-python>`_ version 0.4.6
-       * `gevent <https://pypi.python.org/pypi/gevent>`_ version 1.1.2
-       * `six <https://pypi.python.org/pypi/six>`_ version 1.8.0
+    *   Python and modules.
 
-   To install all required dependencies, follow the instructions for your OS:
+        Python interpreter is not necessary for building Tarantool itself, unless you
+        intend to use the “Run the test suite". For all platforms, this is python
+        version 3.x. You need the following Python modules:
 
-   * For Debian/Ubuntu, say:
+        *   pyyaml
+        *   gevent
+        *   six
 
-     .. code-block:: console
+    To install all required dependencies, follow the instructions for your OS:
 
-        $ apt install -y build-essential cmake make coreutils sed \
-              autoconf automake libtool zlib1g-dev \
-              libreadline-dev libncurses5-dev libssl-dev \
-              libunwind-dev libicu-dev \
-              python python-pip python-setuptools python-dev \
-              python-msgpack python-yaml python-argparse python-six python-gevent
+    *   For Debian/Ubuntu, say:
 
-   * For RHEL/CentOS (versions under 8)/Fedora, say:
+        ..  code-block:: console
 
-     .. code-block:: console
+            $ apt-get install -y \
+              git build-essential cmake make coreutils autoconf automake libtool sed \
```

Comment:
Is there a documentation issue for removing autotools? It is gone [since 2.6.1](
https://github.com/tarantool/tarantool/commit/2b07601922138fee02554ba69c989bee567e812e) (for all 2.7+ as well, but not for 1.10).